tcms/tests/__init__.py

```diff
@@ -229,13 +229,15 @@ def setUpTestData(cls):
         cls.execution_6 = executions[5]
 
 
-class AssignPermissionsToTestUser(LoggedInTestCase):
-    """Assign required permissions to user """
+class PermissionsTestCase(LoggedInTestCase):
+    """Class is used for assigning permissions on test users. """
 
     def remove_all_permissions_except_tested(self, tested_permission):
 
-        for perm in Permission.objects.all():
-            remove_perm_from_user(self.tester, perm)
+        remove_perm_from_user(self.tester, *self.tester.user_permissions
+                              .all())
```

Comment:
When you select all permission via the model manager you can also delete them directly by issuing a DELETE statement against the DB. No need to iterate over all of the objects and delete them one by one.